estate/models/estate_property.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
```

Comment:
imports must be ordered according to https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html#imports